libs/core/tsconfig.lib.json

```diff
@@ -12,7 +12,7 @@
         "experimentalDecorators": true,
         "importHelpers": true,
         "types": [],
-        "lib": ["dom", "es2015"]
+        "lib": ["dom", "es2018"]
```

Comment:
Is there any reason for this change?

Reply:
I want to use `Object.entries`, and don't want to use it without a polyfill. Do we have blockers to update the version?

Also, I saw an issue from @frankfangcn https://github.com/SAP/fundamental-ngx/issues/3710